documentation/modules/overview/con-configuration-points-common.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is included in:
+//
+// overview/assembly-configuration-points.adoc
+
+// UserStory: Common configuration options
+
+[id="configuration-points-common_{context}"]
+= Common configuration
```

Comment:
Yes. I wondered about keeping or removing it. I thought to keep it in just to underline how much can be configured. I think it might be useful to see the whole doc before deciding. Some aspects like logging are useful to know about.